src/seed/reset-db.js

```diff
@@ -1,30 +1,19 @@
-import { query } from '../graphql-schema'
+import { cleanDatabase } from './factories'
 import dotenv from 'dotenv'
-import neo4j from '../bootstrap/neo4j'
 
 dotenv.config()
 
 if (process.env.NODE_ENV === 'production') {
-  throw new Error('YOU CAN`T UNSEED IN PRODUCTION MODE')
+  throw new Error(`YOU CAN'T CLEAN THE DATABASE WITH NODE_ENV=${process.env.NODE_ENV}`)
 }
 
-const driver = neo4j().getDriver()
-const session = driver.session()
-
-const deleteAll = `
-MATCH (n)
```

Comment:
@appinteractive here is the former location of the database cleaner query.

Reply:
Here the relations or the detach was missing.